.github/workflows/archive_build.yml

```
@@ -1,20 +1,29 @@
-        uses: actions/checkout@v3
-        with:
-          submodules: recursive
-          fetch-depth: 0
-      - name: Build Firmware
-        uses: ./.github/actions/build
-        with:
-          BUILD_WRAPPER_OUT_DIR: build_wrapper_out_dir
+      - name: Fetch binary
+        env:
+          firmwareUrl : https://github.com/jpcornil-git/ayabAsync-firmware/releases/download/${{ env.AYAB_VERSION }}/firmware_uno_1_1_1.hex
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:20: Double quote to prevent globbing and word splitting

(shellcheck)